scenes/act1.md

```diff
@@ -339,7 +339,7 @@ publish("hp_show");
 
 `_.whitebread = true`
 
-n: HAI USATO LA *PAURA DI ESSERE FERITI*
+n: HAI USATO LA *PAURA DI FARSI MALE*
```

Comment:
questo dovrebbe essere cambiato in tutte le altre scene

Reply:
"Farsi male" mi sembra più adatto allo stile narrativo, apro una PR per cambiare tutti gli altri,.

Reply:
Fatto un commit per le altre scene